pagefaultgames/pokerogue
# [Sprite] Beta2 Variant Patch icon chores

## What are the changes the user will see?
No user-side changes.

## Why am I making these changes?
#3470 did not update any icons in the `variant` folders. (They should appear after the folder for Gen `9`)
![image](https://github.com/user-attachments/assets/267bb3a5-cb34-4ec0-902e-3b3fac814b92)

On top of various fixes necessitated by that batch of icons, _other_ icons also needed some updates.

Although the icons in these folders aren't user-facing, and do not have an immediate impact, collecting them all in one place is a crucial part of efficiently generating icon sheets.

## What are the changes from a developer perspective?
The images in these folders now match the art team's desired icon style goals more closely.

## How to test the changes?
No changes should be observed in Starter Select, or any other place where icons are used.

## Checklist
~~- [ ] Have I considered writing automated tests for the issue?~~
~~- [ ] If I have text, did I add make it translatable and added a key in the English language?~~
~~- [ ] Are the changes visual?~~
~~  - [ ] Have I provided screenshots/videos of the changes?~~
